pkgs/development/python-modules/notifications-android-tv/default.nix

```
@@ -0,0 +1,32 @@
+{ lib
+, python3
+, fetchPypi
+}:
+
+python3.pkgs.buildPythonPackage rec {
```

Common way for Python modules:

```suggestion
{ lib
, buildPythonPackage
, fetchPypi
}:

buildPythonPackage rec {
```

---

I will update nix-init's style since that seems like the preferred style

---

fixed in https://github.com/nix-community/nix-init/commit/f9aebb3ad301d2d3d25b98648f4737d9ebb5e063, `buildPythonPackage` now declares all the inputs and only works with the python's `callPackage`, `buildPythonApplication` still uses `with python3.pkgs` like it used to

---

@figsoda, thanks.